Cargo.toml

```
@@ -16,6 +16,13 @@ cidr = "0.2"
 lazy_static = "1.4"
 uuid = "1.5"
 regex = "1.10"
+serde = { version = "1.0", features = ["derive"], optional = true }
+serde_regex = { version = "1.1.0", optional = true }
 
 [lib]
-crate-type = ["cdylib", "staticlib"]
+crate-type = ["lib", "cdylib", "staticlib"]
+
+[features]
+default = ["ffi"]
+ffi = []
+serde = ["cidr/serde", "dep:serde", "serde_regex"]
```

```suggestion
serde = ["cidr/serde", "dep:serde", "dep:serde_regex"]
```

? Is `serde_regex` expected?

---

No, I missed that out. I added it back.